lxd/storage_btrfs.go

```diff
@@ -1251,7 +1253,11 @@ func (s *storageBtrfs) btrfsPoolVolumeCreate(subvol string) error {
 	return nil
 }
 
-func (s *storageBtrfs) btrfsPoolVolumeQGroup(subvol string) (string, error) {
+func (s *storageBtrfs) btrfsPoolVolumeCreate(subvol string) error {
+	return btrfsSubVolumeCreate(subvol)
+}
```

Comment:
Can't we do away with those wrapper functions and just call btrfsSubvolumeCreate directly?